blockchain/eth-query-and-execute.go

```diff
@@ -0,0 +1,417 @@
+package blockchain
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/gorilla/websocket"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/external-initiator/store"
+	"github.com/smartcontractkit/external-initiator/subscriber"
+	"github.com/status-im/keycard-go/hexutils"
+)
+
+const (
+	ETH_QAE            = "eth-query-and-execute"
+	defaultResponseKey = "value"
```

Comment:
This field should be called "result" when POSTing data to the CL node.

For example, if I set up an express.js endpoint to see what the EI POSTs, I receive:
```
POST Data:  {
 "value": "0xd012f4feac4e9bf49e6932b8725f003a37630287"
}
```
But this should be:
```
POST Data:  {
 "result": "0xd012f4feac4e9bf49e6932b8725f003a37630287"
}
```

Reply:
This can be set in the params, but yes, the default key should be "result".